rocky/katalogus/templates/plugin_detail.html

```diff
@@ -20,53 +19,49 @@ <h1>{{ plugin.name }}</h1>
                     <p>
                         <strong>{% translate "Description" %}</strong>
                     </p>
-                    <p>
-                        {{ plugin.description }}
-                    </p>
-
+                    <p>{{ plugin.description }}</p>
                     <div class="horizontal-view">
                         {% include "partials/enable_disable_plugin.html" with plugin=plugin %}
+
                     </div>
                 </div>
-
                 <div class="image-square">
                     <div class="image-container">
                         <img src="{% url "plugin_cover" organization_code=organization.code plugin_id=plugin.id %}"
-                            alt="boefje placeholder image"/>
+                             alt="boefje placeholder image"/>
                     </div>
                 </div>
             </div>
         </section>
-        {% if perms.tools.can_scan_organization %}
+        {% if perms.tools.can_view_katalogus_settings %}
             {% include "plugin_settings_list.html" with object_list=object_list plugin=plugin %}
 
         {% endif %}
         <section>
             <div>
-              <h2>{% translate "Consumes" %}</h2>
-
-              {% if plugin.consumes %}
-                <p>
-                    {% blocktranslate trimmed with plugin_name=plugin.name %}
+                <h2>{% translate "Consumes" %}</h2>
+                {% if plugin.consumes %}
+                    <p>
+                        {% blocktranslate trimmed with plugin_name=plugin.name %}
```

Comment:
Was this the work of  a linter? The changes are hard to spot right now.. Don't we enforce indentation levels?

Reply:
Yes @Donnype it is indeed not nice those differences nothing to do with the PR. We still have to decide which linter we are going to use for templates, I am still using djlint